examples/07-advanced-modeling.ipynb

```diff
@@ -8,9 +8,9 @@
       "source": [
         "# Advanced Model Handling in TIAToolbox\n",
         "\n",
-        "<a href=\"https://colab.research.google.com/github/TIA-Lab/tiatoolbox/blob/master/examples/07_example_modeltechniques.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+        "<a href=\"https://colab.research.google.com/github/TIA-Lab/tiatoolbox/blob/master/examples/07_modeltechniques.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

Comment:
@mostafajahanifar please correct the link.

Reply:
done in 733c3a3